book/src/adding-board-support.md

```diff
@@ -86,6 +86,9 @@ sbd-gen generate-ariel boards -o src/ariel-os-boards --mode update
     - `selects`: A [rustc-target](#adding-support-for-a-processor-architecture) module or one of the `cortex-m*` modules if applicable.
 - In `doc/support_matrix.yml`:
   - Add an entry under `chips`, with the laze context and supported features.
+- In `peripherals.md` of the Ariel OS HAL crate for the MCU family:
+  - Add the peripherals taken from `OptionalPeripherals` for each feature/MCU combination you added.
```

Comment:
```suggestion
  - Add the peripherals taken from `OptionalPeripherals` for each feature/MCU combination you added.
  
    This is a stop-gap measure until we find means to declare the provided peripherals and requirements of drivers in a machine readable way, and can generate assignments at build time.
```

Documenting that this is how we do it carries the risk of ossifying the workaround; this line is aimed at keeping all involved aware of that state.

Reply:
Sounds like a good idea thanks.